api/test/models/location/location-data-mapper-impl.test.ts

```diff
@@ -0,0 +1,132 @@
+import { expect } from 'chai';
+import 'mocha';
+import { anyString, anything, capture, instance, mock, reset, verify, when } from 'ts-mockito';
+import { Location } from '../../../lib/models/location';

```

Comment:
Change this to 
```suggestion
import { Location } from '../../../lib/models/location/location';
```
There is something wacky that happens with the index export and causes the constructor to fail.